utils/build/docker/nodejs/express4-typescript/app.ts

```diff
@@ -147,6 +191,19 @@ app.all('/tag_value/:tag/:status', (req: Request, res: Response) => {
   res.status(parseInt(req.params.status) || 200).send('Value tagged');
 });
 
+app.get('/read_file', (req: Request, res: Response) => {
+  const path = req.query['file'];
+  fs.readFile(path, (err: Error, data: Buffer) => {
+    if (err) {
+      console.error(err);
+      res.status(500).send("ko");
+    }
+    res.send(data);
+  });
+});
+
+require("./iast")(app, tracer);
+
```

Comment:
We know this is missing for the typescript app but it's not the role of this PR to fix it @uurien

Reply:
I can revert the typescript changes, would this PR get merged then? Cuz typescript really doesnt seem that important